gateway/platforms/vertx3/vertx3/src/test/resources/io/apiman/gateway/platforms/vertx3/users.ldif

```diff
@@ -1,23 +1,28 @@
 dn: ou=people,o=apiman
+changetype: add
```

Comment:
Can you explain what this is for?

Reply:
The changetype attribute is now explicitly required by the underlying LDAP library used for testing, otherwise it will fail.

changetype tells the LDAP engine what sort of change you want to make. for example, you are adding, deleting, modifying, etc.

See:
https://www.digitalocean.com/community/tutorials/how-to-use-ldif-files-to-make-changes-to-an-openldap-system